### ISSUE TITLE
Not able to add second device/account

### ISSUE THREAD

**Issue by kringloper** [rkokkelk/nl.xseth.wallbox#24]
When I try to add a second device (same wallbox, different account) I get the message 'no new devices found'.

Usecase: seperate accounts to differentiate sessions (two cars) to be able to make session export for company car.

**Comment by rkokkelk** [comment]
Hi, yeah unfortunately this is not how it currently works. Basically the app is for controlling the same wallbox device and not the actual sessions of the car. Basically, in order to identify for devices, the Wallbox device UID is being used to differentate between the various devices. So because you want to add the same wallbox device twice this is not going to work unfortunately. 

Given your use case I don really know on how to implement this with the current API functionality. Could you make share an example of the JSON output from the API between two different loading sessions from your two different accounts. Maybe there is some identifiable information in the API which I can integrate in the current ap. E.g. maybe a specific tag I can set in the trigger action when a car starts charging.

**Comment by kringloper** [comment]
Just to check. By combining the Wallbox ID and the login name/account name as an unique identifier for a device, you will be able to solve my usecase. This will bring the possibility to choose which device (in combination with the linked account) you use to charge. And then it will still work if one wants to install several wallboxes within one account.

Also, I still can use the current app to retrieve status and use the sample homey script to differentiate users. So no hurries ;-)

**Comment by Boechie** [comment]
> Just to check. By combining the Wallbox ID and the login name/account name as an unique identifier for a device, you will be able to solve my usecase. This will bring the possibility to choose which device (in combination with the linked account) you use to charge. And then it will still work if one wants to install several wallboxes within one account.
> 
> Also, I still can use the current app to retrieve status and use the sample homey script to differentiate users. So no hurries ;-)

@kringloper I don't think that adding the same device twice (but with different credentials) will actually solve your need to distinguish between sessions. Bear in mind that  when you connect a Wallbox through a non-owner account, you can not control it (it will give 403 errors).

Currently we are working on a solution to do session accounting in a correct way with only 1 Wallbox device in Homey. We will soon publish a pull request for an extension of this app that will solve your need (as far as I understand it correctly).

**Comment by kringloper** [comment]
> (it will give 403 errors).

This might also explain why some other people experience the 403 error. My wallbox is out of order for the moment, but hopefully i will be able to experience your changes soon.

**Comment by lmeijer66** [comment]
How can I unlock my Wallbox on my Homey device, using a ‘private’ Apple user account on the Wallbox?
